tests/nnstreamer_decoder/runTest.sh

```diff
@@ -58,23 +58,47 @@ callCompareTest testcase3.con.log testcase3.dec.log 3 "Compare for case 3" 0 0
 ## Trigger "decoder not configured"
 gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} videotestsrc ! tensor_converter ! tensor_decoder mode=direct_video ! video/mpeg ! fakesink" 4F_n 0 1 $PERFORMANCE
 
+# Check existence of tensor_decoder flatbuf sub-plugin
+gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} videotestsrc num-buffers=3 ! \
+    videoconvert ! video/x-raw,width=64,height=32,framerate=5/1,format=RGB ! \
+    tensor_converter ! tensor_decoder mode=flatbuf ! fakesink" 5 1 0 0
+
+if  (( ${_fail} > 0 ))
+then
+    echo "Cannot find tensor_decoder flatbuf sub-plugin"
+    report
+    exit
```

Comment:
If flatbuf does not exist, you need to skip flabuf tests only.
You are skipping protobuf tests, too.